frontend/app/src/store/history/actions.ts

```diff
@@ -130,10 +131,9 @@ const ignoreInAccounting = async (
       };
     }
     commit(HistoryMutations.SET_TRADES, {
-      data,
-      found: state.trades.found,
-      limit: state.trades.limit
-    } as Trades);
+      ...state.trades,
+      data
```

Comment:
Since with the pagination we get all our data from the single endpoint, and we don't have to collect the output of many, wouldn't it suffice just to replace the state completely?

Reply:
Oh yeah I completely forgot about the ignoring functionality...